packages/headless/src/user.ts

```diff
@@ -90,6 +90,50 @@ export function useUpdateUser({
   });
 }
 
+export function useUpdateUserPersonalInfo({
```

Comment:
Let's call this hook `useUpdateUser` becuase it just calling `user.update`. The hook with the same name we can call it `useUpdateFullUser` becuase it his multiple endpoints. 

```suggestion
export function useUpdateUser({
```